app/scripts/modules/cloudfoundry/src/common/modalWizard.less

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright 2018 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import (reference) '~core/presentation/less/imports/commonImports.less';
+
+@indicator-size: 14px;
+@indicator-radius: @indicator-size / 2;
+
+v2-modal-wizard,
```

Comment:
This css is in your provider module, but there's nothing on these selectors (i.e., some naming convention) that indicates that you're scoping them down to just the cloud foundry provider. I'm concerned that they're global overrides.

Reply:
I've scoped it to specifically to cloud foundry.

I'm using the react component `WizardModal` from the core module. It wasn't applying the styles from when I pulled in recent changes. I took the CSS from the core library and added `.modal` css path.

I wasn't sure if this was done by design or an oversight. If this is a problem with the core component I can do separate pull request to the core component and remove the css file.